packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.tsx

```diff
@@ -121,17 +124,23 @@ const BpkGraphicPromo = (props: Props) => {
     // clicks and key presses (Enter/Space) for the whole component, as described here:
     // https://developer.mozilla.org/en-US/docs/Web/Accessibility/Keyboard-navigable_JavaScript_widgets
     <div
-      id={contentId}
+      id={contentId || ''}
```

Comment:
```suggestion
      id={contentId}
```

Why are we setting this to an empty string when missing? 🙂

Reply:
I had asked the same here: https://github.com/Skyscanner/backpack/pull/2535#discussion_r949281964

Looks like a commit may have been reverted

Reply:
TS knows contentId can be null, so requires a fallback.

<img width="522" alt="Screenshot 2022-08-23 at 15 54 08" src="https://user-images.githubusercontent.com/25118364/186190811-0d47c9e6-28a8-4708-ac63-0d07a88f020b.png">